php/Terminus/Commands/SshKeysCommand.php

```diff
@@ -63,7 +64,7 @@ public function add($args, $assoc_args) {
     $file = $this->input()->fileName(
       [
         'args'    => $assoc_args,
-        'dir'     => getenv('HOME') . '/.ssh',
+        'dir'     => Configurator::getHomeDir() . '/.ssh',
```

Comment:
If this is a Windows environment, the '/' before '.ssh' could be an issue.  Maybe getHomeDir() should append a trailing slash as part of the function call?